### ISSUE TITLE
Doesn't work with numbers greater than 1000001

### ISSUE THREAD

**Issue by nitantsoni** [samuelmarina/is-even#53]
I hope this project a joke

**Comment by samuelmarina** [comment]
Then make it work, duh

**Comment by j0nimost** [comment]
Can we do Int64 Max?

**Comment by Hocuri** [comment]
Yeah I think we should all be respectful to the author who put in so much effort. But maybe @nitantsoni you can add some more cases? If everyone only contributes 10 cases, we will get up to 80 000 000 000. Be sure to tell all your friends and family to do their part and to spread the word, so that everyone gets to know about this! We got this!

**Comment by coretravis** [comment]
> I hope this project a joke

Even if it wasn't, now it is
